components/elm/cime_config/testdefs/testmods_dirs/elm/gis20kmERP/user_nl_mali

```diff
@@ -0,0 +1,8 @@
+config_velocity_solver = 'FO'
+config_thickness_advection = 'fo'
+config_thermal_solver = 'none'
+config_pio_num_iotasks = 47
+config_pio_stride = 64
```

Comment:
Also, are you sure these are being used?  This implies the test is using 23.5 nodes on pm-cpu, which is definitely not correct for 20km GIS.  Is it possible the pio settings are getting set automatically later, and these mods aren't actually doing anything (and should therefore be removed)?

Reply:
These are old settings that I've been carrying around since Cori, so we should definitely update them to be correct for the respective machines (PM and Chyrs here). Or remove them if they are not needed?

Reply:
@mperego -- this may be where I meant to flag you for further discussion. Thanks.

Reply:
I don't know whether these are used or not. How many partitions are you using for this test?

I think Jerry will soon submit a subsequent PR, so if you want you can postpone to then these performance related issues.

Reply:
@jonbob , do you know how these MPAS PIO namelist settings are handled in E3SM?

Reply:
@jonbob said: these can/should be left out because they will be set automatically